### PR TITLE
Feature/1130 timeline event representation

### PR DESCRIPTION
**Pull request by danielsteinkogler**
## Basics

- [ ] The PR is rebased with current master
- [ ] I added a line to [changelog.md](/doc/changelog.md)
- [ ] Details of what I changed are in the commit messages
- [ ] References to issues, e.g. `close #X`, are in the commit messages and changelog
- [ ] The buildserver is happy

## Checklist

- [ ] I fully described what my PR does in the documentation
- [ ] I fixed all affected documentation
- [ ] I fixed the introduction tour
- [ ] I wrote migrations in a way that they are compatible with already present data
- [ ] I fixed all affected decisions
- [ ] I added automated tests or a [manual test protocol](../doc/tests/manual/protocol.md)
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I translated all strings visible to the user
- [ ] I mentioned [every code or binary](https://github.com/ElektraInitiative/PermaplanT/blob/master/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
- [ ] I created left-over issues for things that are still to be done
- [ ] Code is conforming to [our Architecture](/doc/architecture)
- [ ] Code is conforming to [our Guidelines](/doc/guidelines)
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)
- [ ] Exceptions to any guidelines are documented

## First Time Checklist

- [ ] I have installed and I am using [pre-commit hooks](../doc/contrib/README.md#Hooks)
- [ ] I am using [Tailwind CSS Linting](https://tailwindcss.com/blog/introducing-linting-for-tailwindcss-intellisense)

## Review

- [ ] I've tested the code
- [ ] I've read through the whole code
- [ ] I've read through the whole documentation
- [ ] I've checked conformity to guidelines
- [ ] I've checked conformity to requirements
- [ ] I've checked that the requirements are tested

### CODE REVIEW

**Comment by danielsteinkogler** [comment]
jenkins build please

**Comment by danielsteinkogler** [comment]
jenkins build please

**Comment by danielsteinkogler** [comment]
jenkins build please

**Comment by danielsteinkogler** [comment]
jenkins build please

**Comment by danielsteinkogler** [comment]
jenkins build please

**Comment by danielsteinkogler** [comment]
jenkins build please

**Comment by danielsteinkogler** [comment]
jenkins build please

**Comment by danielsteinkogler** [comment]
jenkins build please

**Comment by danielsteinkogler** [comment]
jenkins build please

File: e2e/features/timeline.feature [hunk 1]
```diff
@@ -16,10 +16,11 @@ Feature: Planting Timeline
         When I change the plants removed date to yesterday
         Then the plant disappears
 
-    Scenario: Unhide a plant by changing the map date by day
-        When I change the plants added date to tomorrow
-        And I change the map date to tomorrow
-        Then the plant appears
+    # TODO Fix Test (#1202)
```
Comment (danielsteinkogler) [inline e2e/features/timeline.feature:19]: i created a left over issue to fix this scenario. i currently don't know why it is failing. When i test it manually it seems to work.

**Comment by markus2330** [comment]
jenkins build please

**Comment by markus2330** [comment]
jenkins build please

**Comment by markus2330** [comment]
jenkins build please

**Comment by markus2330** [comment]
jenkins build please

**Review by markus2330 — CHANGES_REQUESTED** [review]
Thx for the PR! Finally it built!

We did not see any visual representation of events in the timeline. Please add a (manual) test how to get events into the timeline.

**Comment by danielsteinkogler** [comment]
jenkins build please

**Comment by markus2330** [comment]
jenkins build please

**Review by markus2330 — APPROVED** [review]
**Excellent** work, we are very happy :heart:

**Comment by danielsteinkogler** [comment]
jenkins build please

**Pull request merged**